HolyshiftService/Migrations/20231103211511_PasswordLength.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```

Comment:
Something is wrong here. There is only 1 migrations file. We don't have any migration file that adds the whole table.

I think you should revert things and build your db from scratch since it looks like it's broken.

- Make sure all the migrations folder is the same as master.
- Add a new migration.
- Delete your local db and build it again